classes/checkout/CheckoutAddressesStep.php

```diff
@@ -67,7 +67,7 @@ public function handleRequest(array $requestParams = array())
         if (array_key_exists('use_same_address', $requestParams)) {
             $this->use_same_address = (bool) $requestParams['use_same_address'];
             if (!$this->use_same_address) {
-                $this->step_is_current = true;
+                $this->setComplete(true);
```

Comment:
Is this normal this is the only `setComplete` in the method, and it doesn't match the previous code `$this->step_is_current = true;` not `$this->step_is_complete = true`

Reply:
Thanks, you're right!